app/src/main/java/com/kickstarter/ui/adapters/ThanksAdapter.java

```diff
@@ -60,7 +62,7 @@ public ThanksAdapter(final @NonNull Delegate delegate) {
   }
 
   public void takeData(final @NonNull ThanksData data) {
-    setSection(SECTION_SHARE_VIEW, Collections.singletonList(data.getBackedProject()));
+    setSection(SECTION_SHARE_VIEW, Collections.singletonList(Pair.create(Pair.create(data.getBackedProject(), data.getCheckoutData()), data.getUserEmail())));
```

Comment:
Knowing now that the issue was with the project object being empty we could probably change this back to a triple just to see if it works.  not a blocker though